test/integration-tests/release_3_x.test.ts

```diff
@@ -0,0 +1,105 @@
+"use strict";
```

Comment:
This file name will probably get confusing once we release 4.0. How about call this `core.test.ts` and the old one `subnetNames_securityGroupNames.test.ts` or something like that. Basically name the files after the things they are testing

Reply:
In release 4 I think we need to drop support for `subnetNames` and `securityGroupNames`.
And these files will be changed and some of them removed.

But I'm ok to rename it